rpc/backend/blocks.go

```diff
@@ -190,20 +184,12 @@ func (b *Backend) TendermintBlockByNumber(blockNum rpctypes.BlockNumber) (*tmrpc
 // TendermintBlockResultByNumber returns a Tendermint-formatted block result
```

Comment:
refactor to use the `rpcClient` on the `TendermintBlockByNumber` function (L170)